### ISSUE TITLE
prompt-toolkit error between objection and frida 1.2.3

### ISSUE THREAD

**Issue by mpgn** [sensepost/objection#179]
When I trie to launch objection i have this error:

```
 ~/T/ objection                                                                                                  13:26:23
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 578, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 895, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (prompt-toolkit 1.0.15 (/home/bonclay/.local/lib/python3.6/site-packages), Requirement.parse('prompt-toolkit<3.0.0,>=2.0.0'), {'frida-tools'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/bonclay/.local/bin/objection", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3105, in <module>
    @_call_aside
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3089, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3118, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 580, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 593, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 786, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (prompt-toolkit 1.0.15 (/home/bonclay/.local/lib/python3.6/site-packages), Requirement.parse('prompt-toolkit<3.0.0,>=2.0.0'), {'frida-tools'})
```

Version of prompt-toolkit: 1.0.15

Frida want a version of prompt-toolkit >= 2.0.0 and Objection a version <2.0.0

```
 ~/T/ pip3 install 'prompt-toolkit==2.0.8'                                                                       13:29:28
objection 1.4.3 has requirement prompt_toolkit<2.0.0,>=1.0.15, but you'll have prompt-toolkit 2.0.8 which is incompatible.

 ~/T/ pip3 install 'prompt-toolkit==1.0.15' --user                                                               13:29:57
frida-tools 1.2.3 has requirement prompt-toolkit<3.0.0,>=2.0.0, but you'll have prompt-toolkit 1.0.15 which is incompatible.

```

**Comment by mpgn** [comment]
Fixed with Frida-tools version 1.2.2

**Comment by leonjza** [comment]
Apologies for that, I am working to upgrade objection and release 1.5.0 which will include a prompt-toolkit upgrade. The error is actually as a result of a PR I made in `frida-tools` [here](https://github.com/frida/frida-tools/pull/11), resulting in the incompatible dependency issue.

For now, pinning `frida-tools==1.2.2` should fix until I can get objection fixed up as well.

**Comment by leonjza** [comment]
Just fixed up the prompt-toolkit 2.x support in master [here](https://github.com/sensepost/objection/commit/4671941c8dd7eaba54068822512a597866aa2db0). This will land in the 1.5.0 release soon!

**Comment by leonjza** [comment]
Seems like auto completions are a little broken though (didn't notice the failing test locally), so still have to fix that up!

**Comment by leonjza** [comment]
Fixed now in v1.5.0
